modules/nf-core/smoove/main.nf

```diff
@@ -0,0 +1,35 @@
+process SMOOVE {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::smoove=0.2.8" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/smoove:0.2.8--h9ee0642_0':
+        'brentp/smoove' }"
```

Comment:
This container should be on quay.io too, right?

Reply:
We had an issue with the one on quay.io, it was not working. It is still not working now, but at least it is going further. 
We are working on having it working in singularity too.

Reply:
supplying a docker container should also work for singularity. you can use this until the containers are fixed :)